javascript/passwordToggle.js

```diff
@@ -0,0 +1,19 @@
+const togglePassword = document.querySelectorAll('.eye-btn')
```

Comment:
해당 변수명을 보면 꼭 함수같습니다. 어떤 변수 데이터가 저장된 변수인지 알 수 있게 네이밍 해주시면 좋겠습니다.

```suggestion
const passwordEncryptButtonList = document.querySelectorAll('.eye-btn')
```

Reply:
넵!! 네이밍을 좀 더 신경쓰도록 하겠습니다.